Web/Scotty/Action.hs

```diff
@@ -171,9 +174,13 @@ scottyExceptionHandler = Handler $ \case
     text $ T.unwords [ "Failed to parse parameter", k, v, ":", e]
 
 -- | Uncaught exceptions turn into HTTP 500 Server Error codes
-someExceptionHandler :: MonadIO m => ErrorHandler m
-someExceptionHandler = Handler $ \case
-  (_ :: E.SomeException) -> status status500
+someExceptionHandler :: MonadIO m => Options -> ErrorHandler m
+someExceptionHandler Options{verbose} =
+  Handler $ \(E.SomeException e) -> do
+    when (verbose > 0) $
+      liftIO $
+      putStrLn $ "Caught and exception of " <> show (typeOf e) <> ": " <> show e
```

Comment:
Is it intentional that it goes to stdout, not stderr?

Reply:
https://12factor.net/logs recommend stdout, but I'm not sure

Reply:
I don't have a strong opinion; I think stdout is fine too. By the way `Caught and exception of ` looks like a typo

Reply:
Fixed typo